docs/shortfin/llm/user/shortfin_with_sglang_frontend_language.md

```diff
@@ -0,0 +1,224 @@
+# Using `shortfin` with `sglang`
```

Comment:
Can you add sections for

* Current support status
* Prerequisites (hardware / software / etc.)

Maybe some of that already exists in the e2e_llama8b_mi300x doc, but we should be clear here too.

Reply:
Will add these sections early tomorrow morning

Reply:
Added `Current Support Status` and `Prerequisites` section